lib/web/app/transport.go

```diff
@@ -158,11 +158,8 @@ func dialFunc(c *transportConfig) func(ctx context.Context, network string, addr
 		}
 
 		conn, err := clusterClient.Dial(reversetunnel.DialParams{
-			// The "From" and "To" addresses are not actually used for tunnel dialing,
-			// they're filled out with "dummy values" that make logs easier to read and
-			// debug within the reverse tunnel subsystem.
 			From:     &utils.NetAddr{AddrNetwork: "tcp", Addr: "@web-proxy"},
-			To:       &utils.NetAddr{AddrNetwork: "tcp", Addr: fmt.Sprintf("@app-%v", c.app.Name)},
+			To:       &utils.NetAddr{AddrNetwork: "tcp", Addr: reversetunnel.LocalNode},
```

Comment:
Should there be a `reversetunnel.LocalApp` instead?
`LocalNode` means "local SSH server"

Reply:
That's what I originally had, but it was identical to `LocalNode`.

Reply:
Maybe leave a TODO and add `LocalApp` to all the relevant switches in `lib/reversetunnel`.
Even if it's the same code, it's confusing to read here.